src/mono/mono/mini/graph.c

```diff
@@ -228,6 +227,7 @@ mono_draw_graph (MonoCompile *cfg, MonoGraphOptions draw_options)
 #ifdef HAVE_SYSTEM
 	//com = g_strdup_printf ("dot %s -Tpng -o %s.png; eog %s.png", fn, fn, fn);
 	com = g_strdup_printf ("dot %s -Tps -o %s.ps;gv %s.ps", fn, fn, fn);
+	int _i G_GNUC_UNUSED;
```

Comment:
Do you still need `G_GNUC_UNUSED` here?

Reply:
Yes because otherwise we'd get a warning about the variable being assigned but never used.